ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+HOST_PROJECT="openshift-dev-installer"
+HOST_PROJECT_CONTROL_SUBNET="installer-shared-vpc-subnet-1"
+HOST_PROJECT_COMPUTE_SUBNET="installer-shared-vpc-subnet-2"
+HOST_PROJECT_NETWORK="installer-shared-vpc"
+
+cat > "${PATCH}" << EOF
+featureSet: TechPreviewNoUpgrade
+credentialsMode: Passthrough
+platform:
+  gcp:
+    computeSubnet: ${HOST_PROJECT_COMPUTE_SUBNET}
+    controlPlaneSubnet: ${HOST_PROJECT_CONTROL_SUBNET}
+    createFirewallRules: Disabled
```

Comment:
Aha! It looks like the reason masters can't pull ignition is due to the firewall rules (or lack thereof). I was able to reproduce this locally. Not sure if we have permissions but can we set this to `Enabled` (or just omit it, because i think the default is Enabled).